projects/storefrontapp-e2e-cypress/cypress/integration/product-search-product-type-flow.e2e-spec.ts

```diff
@@ -0,0 +1,179 @@
+import { PRODUCT_LISTING } from './../helpers/data-configuration';
+context('Product search product type flow', () => {
+  before(() => {
+    cy.visit('/');
+  });
+
+  describe('Product search', () => {
+    it('should be able to search with specific product type', () => {
+      const resultsTitleSelector = 'cx-breadcrumb h1';
```

Comment:
as we chosen in query, please remove 'Selector' suffix in all occurrences

Reply:
done